src/modules/Profile.rb

```diff
@@ -737,6 +737,7 @@ def checkProfile
         "\n"
       valid = true
 
+      # TODO: Use Yast::XML.validate instead of external xmllint
```

Comment:
BTW the jing_sucks below seems to have been a packaging bug  where a wrapper script did not pass the return code, but in SLE15 it is fixed already.

Why not do it now? Simply out of time or are there yaks to shave?

Reply:
Good news, this is the only caller :)
https://github.com/yast/yast-autoinstallation/blob/master/src/include/autoinstall/conftree.rb#L687